server/src/leihs/procurement/utils/sql.clj

```diff
@@ -49,7 +48,7 @@
 
 (defn select-nest
   [sqlmap tbl nest-key]
-  (helpers/merge-select sqlmap [(types/call :row_to_json tbl) nest-key]))
+  (sql/select sqlmap [[[:row_to_json tbl]] nest-key]))
```

Comment:
i guess one `[]` wrapping suffice

Reply:
this is the minimal setup that works
`(sql/select sqlmap [[:row_to_json tbl] nest-key]))`